en/hosting_guide/deploy.rst

```diff
@@ -0,0 +1,75 @@
+Orchestra deploy on integration environment
+===========================================
+
+This tutorial is going to describe step by step how you can deploy the openOrchestra
+project on the integration environment:
+
+Install Gem and Bundler
+-----------------------
+
+In order to separate all the project of your computer, we use bundler to execute only
+the specified version of capifony and capistrano
```

Comment:
I'm not sure about what "In order to separate all the project of your computer" means